index.html

```diff
@@ -43,7 +43,9 @@
 
 <p>SparkR is an R package that provides a light-weight frontend to use Apache Spark from R. SparkR exposes the Spark API through the <code>RDD</code> class and allows users to interactively run jobs from the R shell on a cluster.</p>
 
-<p><i>NOTE: As of April 2015, SparkR has been <a href="https://github.com/apache/spark/pull/5096">officially merged</a> into Apache Spark and is shipping in an upcoming release (1.4) due early summer 2015.</i></p>
+<p><i>NOTE: As of April 2015, SparkR has been <a href="https://github.com/apache/spark/pull/5096">officially merged</a> into Apache Spark and is shipping in an upcoming release (1.4) due early summer 2015. You can contribute and follow SparkR developments on the Apache Spark mailing lists and [issue tracker](https://issues.apache.org/jira/browse/SPARK/component/12325400).</i></p>
```

Comment:
One thing -- this is unfortunately an HTML file, so the link to the issue tracker needs to be a `<a href=`